riff-raff/app/notification/FailureNotificationContents.scala

```diff
@@ -0,0 +1,61 @@
+package notification
+
+import com.gu.anghammarad.models.{Action, Target}
+import controllers.routes
+import deployment.{NoDeploysFoundForStage, ScheduledDeployError, SkippedDueToPreviousFailure, SkippedDueToPreviousWaitingDeploy}
+import magenta.DeployParameters
+
+import java.util.UUID
+
+object FailureNotificationContents {
```

Comment:
This might work better as a class that takes `prefix`, `teamTargetsSearch` and `riffRaffTargets` as parameters? This would clean up the signatures but might make it worse in other ways I haven't thought of...  just a thought.

Reply:
I've converted to a class as it helps us to avoid passing `prefix` around so much: https://github.com/guardian/riff-raff/pull/614/commits/02f5c5f195018e2363bf6216b1cb27a9be461b1f. As `teamTargetsSearch` and `riffRaffTargets` are only used by one function I've left those as function parameters.